.github/workflows/check.yml

```diff
@@ -2,6 +2,10 @@ name: Check
 
 on: pull_request
 
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
```

Comment:
Having `@` and spaces around it is a bit confusing, makes me think it's also some special expression syntax that does something. 

In the linked documentation example they don't use spaces and use `-` instead to connect context variables, which looks more familiar to me

Is there any specific reason to using `@`?

Reply:
No, there's no specific reason. I copied the format from this example: https://docs.github.com/en/actions/examples/using-concurrency-expressions-and-a-test-matrix

It just makes the string a little prettier 

For example: 

```
Canceling since a higher priority waiting request for 'Tests @ aSemy:feat/embedded_static_libs' exists
```

<img width="767" alt="image" src="https://user-images.githubusercontent.com/897017/215582767-8f51658a-e759-462a-b35e-c2b2cb5f3ff3.png">


But in practice I never ever look at the computed value. The action gets cancelled, and a new one is triggered. So if you want a different format, then go ahead.

Reply:
It does indeed make the log messages look more readable :+1: For some reason I thought the group string was an internal thing only